src/pocketmine/level/generator/normal/Normal.php

```diff
@@ -144,12 +144,12 @@ protected function lookup(float $temperature, float $rainfall) : int{
 						return Biome::BIRCH_FOREST;
 					}
 				}else{
-					//FIXME: This will always cause River to be used since the rainfall is always greater than 0.8 if we
-					//reached this branch. However I don't think that substituting temperature for rainfall is correct given
-					//that mountain biomes are supposed to be pretty cold.
-					if($rainfall < 0.25){
+					//According to the wiki, mountains have a temperature of 0.2-0.3,
+					//so substituting rainfall for temperature was actually pretty accurate
+					//These are all the best estimates for mountain temperature
```

Comment:
This comment is going to make no sense in retrospect, because nobody will know that this was a problem after the bug is fixed.

Reply:
Should I delete it then?

Reply:
That would seem to be the obvious action ..

Reply:
Ight ignore my 2IQ question I'll remove it